### DISCUSSION TITLE
Paste big chunk of text scrambled the output

### DISCUSSION THREAD

**Discussion by Abdillah** [helix-editor/helix#7464] [Troubleshooting]
When pasting from system clipboard, with lots of text, led Helix to output / write text in scrambled way (some chunk out of order). Sometime it even freezes.

Anyone experiencing this?

I don't currently find the right issue to chime in. I'm afraid it is my configuration / stack somehow.
I'm using WezTerm (with kitty mode) and wl-clipboard as backend.

**Comment by dead10ck** [comment]
Could you post a recording?

**Reply by Abdillah** [reply]
Sure..

I copy the source code, paste in helix, and then back to browser to look for string "</115" which never occurred.
I'm sorry forgot to show my cursor.

[Screencast from 2023-06-28 14-19-59.webm](https://github.com/helix-editor/helix/assets/3030950/9fb97251-b145-45a8-96ca-c011c749ee76)

**Reply by dead10ck** [reply]
What version of helix is this?

**Reply by Abdillah** [reply]
A fresh 23.05:

```
$ hx --version
helix 23.05

$ hx --health clipboard
System clipboard provider: wl-paste+wl-copy
```

I'll try using GNOME console to see if it had anything to do with WezTerm.

**Reply by Abdillah** [reply]
Yes, confirmed. Using wezterm cause this issue.
I have tried using no config in wezterm and still has this problem.

**Reply by Abdillah** [reply]
Whoops, someone actually did report this issue https://github.com/wez/wezterm/issues/3683.
I didn't try to search with "helix" keyword before.
I will check the latest build soon
